frontend/locales/en-US.json

```diff
@@ -79,6 +82,7 @@
   },
   "failedRetrievingDisplayPreferences": "Unable to get display preferences. Using last known settings.",
   "failedSettingDisplayPreferences": "Unable to update display preferences.",
+  "failedToDeleteItem": "Failed to delete item",
```

Comment:
We want to remove a lot of useless error message. We still don't have that documented anywhere, but this is roughly what we want:

* When we don't know what caused the error: Use the generic `anErrorHappened` key.
* If we know what the error is: Use a custom message

In this case, we don't add any useful information of **why** the process failed. If we had something relevant to say (i.e lack of permissions, etc), then it would make sense a new error message. But if there's no known way this can fail, we're better off using the generic error message (the user knows anyway that he was deleting an item, no need to tell him again).